var/spack/repos/builtin/packages/py-sphinx-rtd-theme/package.py

```diff
@@ -22,12 +24,9 @@ class PySphinxRtdTheme(PythonPackage):
     version("0.4.3", sha256="728607e34d60456d736cc7991fd236afb828b21b82f956c5ea75f94c8414040a")
 
     depends_on("py-setuptools", type="build")
-
-    depends_on("py-sphinx@1.6:6", when="@1:", type=("build", "run"))
+    depends_on("python@:3.11", when="@:1.0.0")  # distutils was removed in python@3.12
+    depends_on("py-sphinx@1.6:", when="@1:", type=("build", "run"))
     depends_on("py-sphinx@:6", when="@0", type=("build", "run"))
-    depends_on("py-docutils@:0.18", when="@1.2:", type=("build", "run"))
-    depends_on("py-docutils@:0.17", when="@1:1.1", type=("build", "run"))
-    depends_on("py-docutils@:0.16", when="@0.5.2:0", type=("build", "run"))
```

Comment:
Why were these removed for older versions still listed in the package?

Reply:
I mentioned this in the description as potentially controversial.  It appears that previous py-sphinx-rtd-theme recipes were just copying the py-docutils dependencies from the py-sphinx recipe.  That seemed a bit silly to me since this recipe already depends on py-sphinx with the exact same dependencies.  I did some testing to make sure the builds actually work but I would understand if it is desired to be more explicit here and put them back in.

Reply:
If you confirmed all of the affected versions still build, then I am fine with the change. @manuelakuhn What do you think?

Reply:
Key would be if all of the versions build **and run** without the corresponding dependency.

Reply:
I did test that the theme works on builds 0.5.2, 1.2.0, and 3.0.0.  That was how I found the distutils dependency problem with python@3.12.